### PR TITLE
Small updates to aliases and commands

### PR DESCRIPTION
**Pull request by cschuijt**
Added a few aliases to aliases.json, removed unnecessary chat_id declarations from commands, fix a markdown parsing error on the reply when someone tries to do /winrate without any arguments.

### CODE REVIEW

**Comment by cschuijt** [comment]
I've added a fix for #43, a fix for markdown escaping that wasn't working properly before, and the ability for slarkbot to say nice when someone's message contains 69 or 420 :)

File: src/bot/bot_factory.py [hunk 1]
```diff
@@ -41,7 +42,7 @@ def create_bot():
             ["recents", "matches"], user_commands.run_get_player_recents_command
         )
     )
-    dp.add_handler(CommandHandler("help", help_command.run_help_command))
+    dp.add_handler(CommandHandler(["help", "start"], help_command.run_help_command))
```
Comment (connormullett) [inline src/bot/bot_factory.py:45]: I like it

File: src/bot/commands/helpers.py [hunk 2]
```diff
@@ -196,7 +191,7 @@ def create_match_detail_message(match_data):
         output_message += f"{team} | {player_name} | {hero_name} | {kda} | {cs} | {net_worth} | {gpm} | {xpm}\n"
 
     # Escape markdown up to this point
-    output_message = escape_markdown(output_message)
+    output_message = escape_markdown(output_message, version=2)
```
Comment (connormullett) [inline src/bot/commands/helpers.py:194]: Very nice

File: src/constant_data/aliases.json [hunk 3]
```diff
@@ -35,6 +35,10 @@
     "id": 17,
     "aliases": ["Storm"]
   },
+  {
+    "id": 19,
+    "aliases": ["Tony", "Luka"]
```
Comment (connormullett) [inline src/constant_data/aliases.json:40]: lmfao

**Review by connormullett — APPROVED** [review]
Looks good. Sorry about delay. Approving and releasing as minor

**Pull request merged**